app/scripts/modules/core/src/pipeline/config/stages/googleCloudBuild/GoogleCloudBuildStageForm.tsx

```diff
@@ -50,7 +55,10 @@ export class GoogleCloudBuildStageForm extends React.Component<
   };
 
   private onAccountChange = (accountOption: Option) => {
-    this.props.formik.setFieldValue('account', accountOption.value);
+    const account = accountOption.value;
+    this.props.formik.setFieldValue('account', account);
+    // OptionValues can also be number or boolean, but in this case, account is always a string.
+    this.props.fetchGcbTriggers(account as string);
```

Comment:
We won't need the `as` operator if we parameterize the `Option` type on line 57:

```
private onAccountChange = (accountOption: Option<string>) => {
```